lms/static/sass/xblocks/poll_survey.scss

```diff
@@ -33,11 +35,29 @@ body.new-theme {
                 }
             }
 
+            .poll-results {
+                padding-left: 1em;
+
+                poll-top-choice {
+                    color: #2042CB;
+                }
+
+                .percentage-gauge-container {
+                    .percentage-gauge {
+                        background-color: #EEEEEE;
```

Comment:
@prakashpathak02 We have variables for color. Please use them everywhere and make sure the variable.scss aligned with mcka-apros/static_v2/scss/common/_variables.scss as that file has all the colors of new UI